src/main/webapp/app/exam/participate/exercises/file-upload/file-upload-exam-submission.component.html

```diff
@@ -1,18 +1,22 @@
-<!--resizable container-->
 @if (exercise) {
+    <h3 class="text-align-left fw-normal">
+        <span>
+            {{ exercise.title }}
+        </span>
+        <!-- prettier-ignore -->
+        <span left-header>&nbsp;({{ exercise.maxPoints }} {{ 'artemisApp.examParticipation.points' | artemisTranslate }}@if (exercise.bonusPoints) {
+            <span
+            >, {{ exercise.bonusPoints }} {{ 'artemisApp.examParticipation.bonus' | artemisTranslate }}</span>
+            }) @if (exercise.includedInOverallScore !== IncludedInOverallScore.INCLUDED_COMPLETELY) {
+            <jhi-included-in-score-badge
+            [includedInOverallScore]="exercise.includedInOverallScore" />
+          }</span>
+    </h3>
+    <hr />
+    <!--resizable container-->
     <jhi-resizeable-container [examTimeline]="examTimeline">
```

Comment:
I don't think the comment is needed

```suggestion
    <jhi-resizeable-container [examTimeline]="examTimeline">
```